### ISSUE TITLE
error while running training code.txt mentioned in fifth step

### ISSUE THREAD

**Issue by mkthuria29** [Spidy20/Music_player_with_Emotions_recognition#1]
![Screenshot (1)](https://user-images.githubusercontent.com/29831120/57872471-63828080-7829-11e9-9af9-198164c874eb.png)

**Comment by Spidy20** [comment]
you have to install tensorflow library than it will work.

**Comment by mkthuria29** [comment]
Thanks for that. I’ve done that. I just can’t figure out how to give path of songs folder if u can help

**Comment by Spidy20** [comment]
@mkthuria29 hey see in program where I define If conditions 
In that place I defined my songs folder path 
But you have to copy your songs folder path and you have to past it in code 
It's simple 
But it's required some basic programming skill

**Comment by mkthuria29** [comment]
I think u have written “+random file”. So do I need to replace that?

**Comment by Spidy20** [comment]
No

**Comment by mkthuria29** [comment]
I've changed all the paths in this file according my laptop.
Can u plz tell me what's still troubling me?

**Comment by Spidy20** [comment]
please give me error code than i can give you a suggestions

**Comment by mkthuria29** [comment]
Thanks for ur help but I fixed it. Now the only issue I’m having is  the accuracy problem. It sometimes guesses smiling face as an angry face. What to do to improve accuracy?

**Comment by Spidy20** [comment]
Accuracy depends on image quality and your gpu power , we can' to anything ,
the other way is retrain a model with good quality of images

**Comment by mkthuria29** [comment]
can u plz tell me which algorithm u followed? like what model have u used
in tensor flow?

**Comment by Spidy20** [comment]
I used Mobilenet model
